src/me/corriekay/pokegoutil/DATA/managers/AccountManager.java

```diff
@@ -7,256 +7,184 @@
 import com.pokegoapi.auth.PtcCredentialProvider;
 import com.pokegoapi.exceptions.LoginFailedException;
 import com.pokegoapi.exceptions.RemoteServerException;
-import javafx.scene.control.Alert;
-import javafx.util.Pair;
+import me.corriekay.pokegoutil.DATA.enums.LoginType;
+import me.corriekay.pokegoutil.DATA.models.LoginData;
+import me.corriekay.pokegoutil.DATA.models.LoginResult;
 import me.corriekay.pokegoutil.utils.ConfigKey;
 import me.corriekay.pokegoutil.utils.ConfigNew;
 import okhttp3.OkHttpClient;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-
 /*this controller does the login/log off, and different account information (aka player data)
  *
  */
 public final class AccountManager {
 
-    private static final AccountManager S_INSTANCE = new AccountManager();
-    private static boolean sIsInit = false;
-    private static ConfigNew config = ConfigNew.getConfig();
+    private static AccountManager S_INSTANCE;
+
+    public static AccountManager getInstance() {
```

Comment:
Why add a getInstance? Can't you just make static functions, using the S_INSTANCE, instead of saving the instance as a local variable, then calling public non static functions on them?